picard/ui/options/interface.py

```diff
@@ -208,5 +210,19 @@ def starting_directory_browse(self):
             path = os.path.normpath(path)
             item.setText(path)
 
+    def multi_selection_warning(self):
+        if self.ui.toolbar_multiselect.isChecked():
+            dialog = QtWidgets.QMessageBox(
+                QtWidgets.QMessageBox.Icon.Warning,
+                _('Option Setting Warning'),
+                _(
+                    'Enabling the multiple directories option setting may result in external drives not being recognized by Picard.\n\n'
```

Comment:
I wouldn't make it so specific. It's not really that it does not show external drives per se.

I would rather write e.g.: "When enabling the multiple directories option setting Picard will no longer use the system file picker for selecting directories. This may result in reduced functionality."